README.md

```diff
@@ -75,18 +75,22 @@ Please find all important information here:
 
 1. Clone the repository
 
-`git clone git@github.com:xFAANG/askql.git`
+   `git clone git@github.com:xFAANG/askql.git`
 
-2. Install dependencies:
+2. It is advised to install autoreconf as it is used by one of the Node packages used by AskScript Playground.
```

Comment:
What is autoreconf? Does it apply only to Linux? If so maybe, maybe we can explain that in the beginning or make this step applicable only for Linux development

Reply:
The next line is stating Ubuntu/Debian.  What else would you add here?

Reply:
It's not clear to me if I need to think about `autoreconf` if I'm on MacOS or Windows